extensions/network/net-network-manager.sh

```diff
@@ -0,0 +1,47 @@
+#
+# Extension to manage network interfaces with NetworkManager + Netplan
+#
+function add_host_dependencies__install_network_manager() {
```

Comment:
`add_host_dependencies` is not adequate for this. use `extension_prepare_config` or such.